generators/server/templates/build.gradle.ejs

```diff
@@ -54,7 +54,6 @@ plugins {
 apply plugin: 'java'
 sourceCompatibility=<%= JAVA_VERSION %>
 targetCompatibility=<%= JAVA_VERSION %>
-// Until JHipster supports JDK 9
 assert System.properties['java.specification.version'] == '1.8'
```

Comment:
isn't this line preventing a project to built by gradle, if you try this with JDK 11?

Reply:
Oh you are correct!!!! Let me have a look

Reply:
OK I'm fixing this in 2 minutes